lib/ansible/plugins/doc_fragments/connection_persistent.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment(object):
+
+    # Standard files documentation fragment
+    DOCUMENTATION = r'''
+options:
+  persistent_connect_timeout:
```

Comment:
If the goal is to make this reusable in other persistent connections, that sounds fine, but `persistent_connect_timeout` doesn't need to be in every plugin (`httpapi` in particular has no independent connect step to time out from) and should probably be returned to `network_cli`